Vaibhav Shukla

```diff
@@ -0,0 +1 @@
+
```

Comment:
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Styled Webpage</title>
    <style>
        /* Styling for the headings */
        h1 {
            font-size: 3em;
            color: red; 
            font-family: 'Arial', sans-serif;
        }
        h2 {
            font-size: 2.5em;
            color: skyblue;
            font-family: 'Georgia', serif;
        }
        h3 {
            font-size: 2em;
            color: green;
            font-family: 'Courier New', monospace;
        }

        /* Styling for paragraphs */
        p#para1 {
            font-family: 'Times New Roman', serif;
            font-size: 1.2em;
            color: violet;
        }
        p#para2 {
            font-family: 'Verdana', sans-serif;
            font-size: 1.5em;
            color: purple;
        }
        p#para3 {
            font-family: 'Courier New', monospace;
            font-size: 1.1em;
            color: black;
        }
    </style>
</head>
<body>

    <h1>This is the Main Heading (h1)</h1>
    <h2>This is a Subheading (h2)</h2>
    <h3>This is a Smaller Subheading (h3)</h3>

    <p id="para1"><strong>This paragraph</strong> uses the <em>Times New Roman</em> font and has a <strong>bold</strong> and <em>italic</em> style. The text color is dark blue, and the font size is slightly larger.</p>
    
    <p id="para2"><em>This second paragraph</em> has a different <strong>Verdana</strong> font, a larger font size, and a dark magenta color. It also contains <strong>bold</strong> and <em>italic</em> formatting.</p>

    <p id="para3">The <strong>third paragraph</strong> is written in <em>Courier New</em> with a golden color. It uses <strong>bold</strong> and <em>italic</em> for emphasis and has a different size compared to the others.</p>

</body>
</html>